### ISSUE TITLE
Flags

### ISSUE THREAD

**Issue by dkhamsing** [dkhamsing/open-source-ios-apps#585]
It was brought to our attention that the flag we are using for languages actually misrepresent the project: ie Spanish/Mexico, Chinese/Taiwan 

Let’s brainstorm some ideas including removing the flags if they are not useful

**Comment by vpeschenkov** [comment]
I don't think the flag is useful for the list. As I got, we don't have any acceptable solution. 

The field is used 41 times when the list has 814 items (5%). In my point of view, flag refers to the country; not to the language. I think it does not matter which language is used in the app, but it may be nice to know in which country it was developed. I suggest replacing this field with a `country` field. However, as we don't have anything to represent it as something useful (e.g., showing only the apps for a country), I think it also is a bad idea.

So, I propose to remove this field.

**Comment by shunchu** [comment]
Knowing an app is of a particular language is certainly "good to have" but I don't think it'd make or break the usefulness of the list. The notion of "flags" useful visual candy but can be distracting for the purpose of what the list is meant to serve (showcase open source iOS apps). Mexican apps probably don't want to necessarily be associated with a Spanish flag. And it gets unnecessarily controversial (for some) when a Chinese flag is stamped on a Taiwanese app or vice versa. It just gets too complicated and does this list a disservice IMO.

**Comment by vpeschenkov** [comment]
No, I don't. But for me can be interesting to know in which country it was (e.g., all the apps what was made in Australia). And we miss one point that an app can be in several languages. So, in this case, we should present them all. Also, in our case, it often means that README is not in English.

BTW, I've thought how to show only the fact what an app is not in English, but I couldn't think anything helpful up.

**Comment by vpeschenkov** [comment]
Also, according to this article https://en.wikipedia.org/wiki/Flag:

> As languages rarely have a flag designed to represent them,[13] it is a common but unofficial practice to use national flags to identify them. The practice is discouraged[14] and can be irritating[15][16] and because flags tend to evoke feelings other than the intended meaning.

**Comment by vpeschenkov** [comment]
@dkhamsing If you think it is useful to show the fact that an app is not in English, we can do it. But just in case to show the fact of non-English. I guess we can use a flag from here https://emojipedia.org/flags/ or any other suggestions.

**Comment by dkhamsing** [comment]
Definitely overlooked the sensitive nature of nationality - let’s move to remove the flags for now - looks like we should show when an app is not in English later 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by dkhamsing** [comment]
Flags were removed - thx everyone cc @ethanhuang13

**Comment by dkhamsing** [comment]
Please vote  👍 for the options below to represent `not English`

**Comment by dkhamsing** [comment]
🌐

**Comment by dkhamsing** [comment]
🌎

**Comment by scribblemaniac** [comment]
I definitely think we should include the field, even if it's not being displayed in the readme or is only being displayed in the dropdown information as "Language(s): ...". It could be used later to filter the apps by language. Certainly this is of interest to some people because it is very difficult to use an app that is in a language you don't understand.

**Comment by vpeschenkov** [comment]
> Edit: If we use a symbol like 🌐 , then it could be used to represent not-spanish for a spanish translation of the readme, and so on, if we ever get that (see #554).

I don't think that somebody will be able to handle and maintain the list localization. Moreover, I don't see a sense of that 😞.
